README.md

```diff
@@ -49,6 +49,23 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [official TiDB blog](https://www.pingcap.com/blog/).
 
+
+## Quick start
+
+### To start using TiDB
```

Comment:
Use TiDB

Reply:
It was referred from k8s https://github.com/kubernetes/kubernetes#to-start-using-kubernetes